package.json

```diff
@@ -40,6 +40,7 @@
     "sinon": "4.5.0",
     "sinon-chai": "3.0.0",
     "standard": "11.0.1",
+    "timekeeper": "^2.1.2",
```

Comment:
use the exact version. Add an npmrc file like the broker does https://github.com/kinesis-exchange/broker/blob/master/.npmrc